backend/test/dotenv-config.ts

```diff
@@ -1,3 +1,3 @@
 require('dotenv').config({
-  path: '.env.test',
+	path: '.env.test',
```

Comment:
not sure what happens here? (surely it is corretct, but I'm curious!)

Reply:
this is pretterjs + some config changing the spaces to a tab

Reply:
thanks